packages/common-all/src/timing.ts

```diff
@@ -18,3 +18,13 @@ export const nanosToMillis = (nanos: number) => {
 export const milliseconds = () => {
   return new Date().getTime();
 };
+
+export class TimeUtils {
+  static async sleep(ms: number) {
```

Comment:
I feel like I remember seeing @tma66 add something similar recently?  Was it in test-utils though? Maybe we can consolidate that version to this one.